bin/validate.js

```diff
@@ -113,16 +109,38 @@ function validatePayloadCodecs(vendorId, payloadEncoding) {
     }
   });
 
-  runs.forEach((r) => {
-    promises.push(
-      new Promise((resolve, reject) => {
-        exec(
-          `bin/runscript -codec-path "${r.fileName}" -routine ${r.routine} -input '${JSON.stringify(r.input)}'`,
-          (err, stdout, stderr) => {
-            if (err) {
-              reject(err);
-            } else if (stderr) {
-              reject(stderr);
+  for (let index = 0; index < runs.length; index++) {
+    const r = runs[index];
+    await new Promise((resolve, reject) => {
+      setTimeout(async () => {
```

Comment:
You can remove this

Reply:
I've made a change that should hopefully be what your intention is.